src/app.rs

```diff
@@ -716,10 +740,12 @@ mod tests {
             config: Some(Config {
                 using_region: Some(String::from("ap-northeast-1")),
                 using_table: Some(String::from("cfgtbl")),
+                using_port: Some(8000),
             }),
             cache: None,
             overwritten_region: None,
             overwritten_table_name: None,
+            overwritten_port: None,
             output: None,
         };
         assert_eq!(cx2.effective_region(), Region::from_str("ap-northeast-1")?);
```

Comment:
Could you write a new test for the local region? If you are busy, I can take over writing tests for the local region.

Reply:
Thanks for feedback.
I'll try to add `--region local`'s test.